day_01.md

```diff
@@ -128,8 +130,10 @@ Well, by the time being we're on the [ES2019 version](https://www.ecma-internati
    3. [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
 4. [Introduction to NPM](https://docs.npmjs.com/about-npm/)
 
+---
```

Comment:
Use *** for horizontal rule, it's just for consistency (if you consider 3 dashes is better then let's review all pages to change them accordingly)